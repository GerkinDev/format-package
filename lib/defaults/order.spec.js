const order = require('./order');

describe('default order', () => {
  it('can be loaded', () => {
    expect(order).toMatchSnapshot();
  });

  it('has a `...rest` key', () => {
    expect(order.indexOf('...rest')).not.toEqual(-1);
  });
});
