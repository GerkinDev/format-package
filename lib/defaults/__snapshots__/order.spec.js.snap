// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`default order can be loaded 1`] = `
Array [
  "name",
  "version",
  "description",
  "license",
  "private",
  "engines",
  "os",
  "cpu",
  "repository",
  "bugs",
  "homepage",
  "author",
  "keywords",
  "bin",
  "man",
  "main",
  "module",
  "browser",
  "files",
  "directories",
  "config",
  "publishConfig",
  "scripts",
  "husky",
  "lint-staged",
  "...rest",
  "dependencies",
  "peerDependencies",
  "devDependencies",
  "optionalDependencies",
  "bundledDependencies",
]
`;
